app.js

```diff
@@ -89,6 +89,21 @@ const listingManager = require('lib/bptf-listings');
 
 log.info(package.name + ' v' + package.version + ' is starting...');
 
+const request = require('@nicklason/request-retry');
+const semver = require('semver');
+setInterval(function () {
+	request.get('https://raw.githubusercontent.com/Nicklason/tf2-automatic/master/package.json', function (err, body) {
+		if (err) {
+			log.warn('Failed to check for updates: ' + err);
+			return
+		}
+
+		if (semver.lt(package.version, body.version)) {
+			require('app/admins').message('Update available! Download it here: https://github.com/Nicklason/tf2-automatic');
+		}
+	});
+}, 1000 * 60 * 60);
+
```

Comment:
Create a file inside the app folder called "version-check.js" and just require it on startup.

Actually would be a good idea to have a function inside that file called checkUpdates and call it as soon as the file is required.